doc/ko/uftrace-live.md

```diff
@@ -709,6 +716,24 @@ uftrace 를 아래의 예제에서 평소처럼 사용할때에는 에러 메세
        2.451 us [19387] |   } /* a */
        3.289 us [19387] | } /* main */
 
+`-U` 옵션은 `-P` 옵션과 반대의 효과가 있어 사용자가 두 옵션 모두를 제어할 수 있다.
+나중에 명시된 옵션은 앞선 옵션을 덮어쓴다.
+예를 들어, 위의 예시에서 'a'를 제외한 모든 함수들을 추적하고 싶다면 아래와 같이 옵션을 적용한다.
+
+    $ uftrace --no-libcall -P . -U a  abc
+    # DURATION    TID     FUNCTION
+                [19390] | main() {
+                [19390] |   b() {
+       0.983 us [19390] |     c();
+       2.012 us [19390] |   } /* b */
+       3.373 us [19390] | } /* main */
+
+옵션의 순서는 중요하다.  만일 옵션의 순서를 `-U a -P .`와 같이 변경한다면
+`-P .` 옵션이 전체에 유효해지기에 모든 함수들을 추적할 것이다.
+
+추가적으로, `-U` 옵션은 `-pg` (그리고 `-mfentry` 혹은 `-mrecord-mcount`)로 빌드된 바이너리 내의
+함수들을 비활성화하는데 사용될 수 있다.  이 때, capstone이 필요할 수 있다.
+
```

Comment:
original:

```
Note that `-U` option has the opposite effect of `-P` option so users can
the both to fine-control.  The option comes later will override the formers.
For example if you want to trace all functions but 'a' in the above:

    $ uftrace --no-libcall -P . -U a  abc
    # DURATION    TID     FUNCTION
                [19390] | main() {
                [19390] |   b() {
       0.983 us [19390] |     c();
       2.012 us [19390] |   } /* b */
       3.373 us [19390] | } /* main */

The order of the options is important, if you change it like `-U a -P .` then
it will trace all the functions since `-P .` will be effective for all.

In addition, the `-U` option can be used to disable functions in binaries
built with `-pg` (and `-mfentry` or `-mrecord-mcount`).  It might require
capstone to parse the instructions.
```